Source/JavaScriptCore/runtime/ImportMap.cpp

```diff
@@ -265,7 +265,7 @@ Expected<void, String> ImportMap::registerImportMap(const SourceCode& sourceCode
 
 String ImportMap::integrityForURL(const URL& url) const
 {
-    return m_integrity.get(url);
+    return !url.isNull() ? m_integrity.get(url) : String();
```

Comment:
Usually this kind of thing reads better without a `!` but I suppose it's a matter of taste.

Reply:
Ok, will fix. I hesitated.